instrumentation/netty/netty-3.8/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v3_8/server/HttpServerRequestTracingHandler.java

```diff
@@ -46,7 +46,7 @@ public void messageReceived(ChannelHandlerContext ctx, MessageEvent event) {
     HttpRequest request = (HttpRequest) message;
 
     Context context =
-        tracer().startSpan(request, ctx.getChannel(), channelTraceContext, "netty.request");
+        tracer().startSpan(request, ctx.getChannel(), channelTraceContext, request.getMethod() + " " + request.getUri());
```

Comment:
we need to follow OpenTelemetry semantic conventions (see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md#name)

```suggestion
        tracer().startSpan(request, ctx.getChannel(), channelTraceContext, "HTTP " + request.getMethod());
```

if you want a more specific operation name including the URI, check out https://docs.microsoft.com/en-us/azure/azure-monitor/app/java-standalone-upgrade-from-2x#operation-names

Reply:
Hi @trask , I need to have more details in my insights. With my proposition I get something like this in the operation name : 
GET /api/v3/tracks

Before that, all calls were grouped by « netty.request » name.
With your proposition, I’ll have something like that : 
HTTP GET
HTTP POST

(For the upstream PR, no pb)